atd-toolbox/data_model/python/40_make_lookup_seeds.py

```diff
@@ -15,8 +16,8 @@ def escape_single_quotes(input_string):
 veh_direction_of_force_lkp_values = """insert into lookups.veh_direction_of_force_lkp (id, label)
     select veh_direction_of_force_id, veh_direction_of_force_desc from atd_txdot__veh_direction_of_force_lkp;"""
 
-mode_category_lkp_values = """insert into lookups.mode_category_lkp (id, label)
-    select  id, atd_mode_category_mode_name from atd__mode_category_lkp;"""
+mode_category_lkp_values = """insert into lookups.mode_category_lkp (id, label, source)
+    select  id, atd_mode_category_mode_name, 'vz' from atd__mode_category_lkp where id != 10;"""
```

Comment:
making it super clear that this lookup table is completely custom by adding a `source` column and setting it to `vz`. (we use the `source` column in other lookup tables which mix CRIS + VZ values—we have a documentation todo about this 👍 )